src/components/Modal.js

```diff
@@ -52,13 +52,14 @@ export default class Modal extends Component {
     onRequestClose: PropTypes.func,
     closeTimeoutMS: PropTypes.number,
     ariaHideApp: PropTypes.bool,
-    shouldFocusAfter: PropTypes.bool,
+    shouldFocusAfterRender: PropTypes.bool,
```

Comment:
This was misnamed because there is no variable called shouldFocusAfter

Reply:
This is really a mismatch between Modal and ModalPortal. You can open a new PR for this one.

Reply:
Ok I'll do that separately. In the mean time, I'll change it back.